dash-renderer/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dash-renderer",
-  "version": "1.8.3",
+  "version": "1.9.0",
```

Comment:
Did we already note the renderer update in the CHANGELOG?

Reply:
Updated "Dash" to "Dash and Renderer" in https://github.com/plotly/dash/pull/1535/commits/6de7f9c749a648079f19fb3a2c331357953c4a43